src/components/forms/KpiAdminForm.vue

```diff
@@ -113,29 +113,6 @@
         />
       </toggle-button>
 
-      <toggle-button v-model="localKpi.api">
```

Comment:
I og med at denne tas ut; fjerne `apiCurl`-metoden og default verdi som settes for `localKpi.api`?

Reply:
:+1: Glemte den.